sui-execution/v1/sui-verifier/src/one_time_witness_verifier.rs

```diff
@@ -28,7 +28,7 @@ use sui_types::{
     base_types::{TX_CONTEXT_MODULE_NAME, TX_CONTEXT_STRUCT_NAME},
     error::ExecutionError,
     move_package::{is_test_fun, FnInfoMap},
-    SUI_FRAMEWORK_ADDRESS,
+    BRIDGE_ADDRESS, SUI_FRAMEWORK_ADDRESS,
```

Comment:
Should we be changing this in v1 and v0? I feel like we shouldn't need to but maybe I'm missing something

Reply:
you are right, we don;t need to change v0 and v1